src/js/index.js

```diff
@@ -0,0 +1,94 @@
+const digitBtn = document.getElementById('num');
+const totalInput = document.getElementById('total');
+const resetBtn = document.getElementsByClassName('modifier')[0];
+const operatorBtn = document.getElementById('operator');
+
+digitBtn.addEventListener('click', function (e) {
+  removeFirstZero();
+  const { nodeValue } = e.target.firstChild;
+
+  if (!isMaxLength()) {
+    totalInput.innerText += nodeValue;
+  } else {
+    alert('숫자는 세 자리까지만 입력 가능합니다!');
+  }
+});
+
+resetBtn.addEventListener('click', function () {
+  totalInput.innerText = 0;
+});
+
+operatorBtn.addEventListener('click', function (e) {
+  removeFirstZero();
+  const { nodeValue } = e.target.firstChild;
+  const { textContent } = totalInput;
```

Comment:
이름을 지어줘 보시면 어떨까요? textContent라는 이름은 일반적으로 사용되기 때문에, 코드를 읽을때 이게 어떤 textContent인지 한 번 더 생각해야할 것 같아요

```suggestion
  const { textContent: total } = totalInput;
```